weave/guardrails/prompt_injection_guardrail.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+import weave
+from weave.guardrails.prompts import (
+    PROMPT_INJECTION_GUARDRAIL_SYSTEM_PROMPT,
+    PROMPT_INJECTION_SURVEY_PAPER_SUMMARY,
+)
+from weave.scorers.llm_scorer import InstructorLLMScorer
+from weave.scorers.llm_utils import OPENAI_DEFAULT_MODEL, create
+from weave.scorers.utils import stringify
+
+
+class LLMGuardrailReasoning(BaseModel):
+    injection_prompt: bool
+    is_direct_attack: bool
+    attack_type: Optional[str]
+    explanation: Optional[str]
+
+
+class LLMGuardrailResponse(BaseModel):
+    safe: bool
+    reasoning: LLMGuardrailReasoning
+
+
+class PromptInjectionLLMGuardrail(InstructorLLMScorer):
```

Comment:
this looks good - i just want to warn you that the `InstructorLLMScorer` accepts a `client` which is not a portable object, meaning anything written in this style will not be executable by the backend (ie. executing scores in the UI or in monitoring contexts)